reduction/lr_reduction/template.py

```diff
@@ -7,8 +7,8 @@
 
 import mantid.simpleapi as api
 import numpy as np
-from mantid.api import *
-from mantid.kernel import *
+from mantid.api import *  #noqa: F403
```

Comment:
Instead of this import all, can you import the module (`from mantid import  api as mapi`) and prefix the algorithms used (e.g `mapi.LoadeventNexus`).
Same with `mantid.kernel` below.